evals/gaia/gaia.py

```diff
@@ -0,0 +1,93 @@
+from huggingface_hub import snapshot_download
+from pathlib import Path
+from inspect_ai import task
+from inspect_ai.scorer import Score, match
+from inspect_ai.solver import basic_agent, system_message, TaskState
+from inspect_ai.tool import web_search, bash
+from inspect_ai.scorer import scorer, Target
+from inspect_ai.dataset import Sample, hf_dataset
+from typing import Callable, Literal
+from inspect_ai import Task
+from inspect_ai.dataset._dataset import FieldSpec
+import os
+from typing import Sequence
+
+# TODO: The constants actaully make thins a bit uglier, I'd love it if people were like "wow look at how clean it is to onboard the dataset"
+DEFAULT_INPUT_PROMPT = """Please answer the question below. You should:
```

Comment:
I think the constatants at the start make it a big uglier...

Reply:
I think that the problem is this prompt gives information that you basically need to complete the eval, and people (or at least we) write their agents assuming that task.input varies (and has all the information needed to complete the task), but their system prompts etc. are constant.

Reply:
Could that just be a system prompt then? (which will end up before the user input)

Reply:
Well we wouldn't want to override incoming user agents system prompt. Take, for example, our system prompt [here](https://github.com/AI-Safety-Institute/agent_framework/blob/b12d536e2c0a538aee2c22ac61cbd6872ad3d16f/projects/agents/agents/base_aisi_agent.py#L34-L41):

it is completley generic, and we optimise it across all tasks (seperately, we might customize one for each task - thats our own thing). we read task.input for how to actually complete the task. Therefore DEFAULT_INPUT_PROMPT needs to have everything that is needed to complete the task.

Reply:
Okay, I get that. If the goal is to have it appear below the task then maybe the task param should just take `None` and then we apply the default inside the function body? (as opposed to in the signature)